components/apimgt/org.wso2.carbon.apimgt.micro.gateway/org.wso2.carbon.apimgt.micro.gateway.tenant.initializer/src/main/java/org/wso2/carbon/apimgt/micro/gateway/tenant/initializer/listener/ServerStartupListener.java

```diff
@@ -82,41 +82,47 @@ private static void initializeTenant() throws Exception {
         String username = config.getFirstProperty(APIConstants.API_KEY_VALIDATOR_USERNAME);
         char [] password = config.getFirstProperty(APIConstants.API_KEY_VALIDATOR_PASSWORD).toCharArray();
         String tenantDomain = MultitenantUtils.getTenantDomain(username);
-        String email = MultitenantUtils.getTenantAwareUsername(username);
-        try {
-            CommonUtil.validateEmail(email);
-        } catch (Exception e) {
-            if (log.isDebugEnabled()) {
-                log.debug("Invalid email is provided: " + email);
-            }
-            email = TenantInitializationConstants.DEFAULT_EMAIL;
-        }
-        if (CommonUtil.isDomainNameAvailable(tenantDomain)) {
-            tenantInfoBean.setActive(true);
-            tenantInfoBean.setAdmin(email);
-            tenantInfoBean.setAdminPassword(password.toString());
-            tenantInfoBean.setFirstname(TenantInitializationConstants.DEFAULT_FIRST_NAME);
-            tenantInfoBean.setLastname(TenantInitializationConstants.DEFAULT_LAST_NAME);
-            tenantInfoBean.setTenantDomain(tenantDomain);
-            tenantInfoBean.setEmail(email);
+        if (!MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equals(tenantDomain)) {
+            String email = MultitenantUtils.getTenantAwareUsername(username);
```

Comment:
What is the rationale behind this logic. In super tenant space tenant aware user name is email address? Shouldn't we check for enable email user name enabled here before make this decision?